frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingBounty.tsx

```diff
@@ -607,9 +607,12 @@ function MobileView(props: CodingBountiesProps) {
                           fontFamily: 'Barlow',
                           marginLeft: '30px'
                         }}
-                        onClick={(e: any) => {
+                        onClick={async (e: any) => {
                           e.stopPropagation();
```

Comment:
@tobi-bams  can you please put all these function calls on the Button onClick function to a new function, then call the function on onClick.

it helps with speed and will make the code tidy, thanks.